MainRepository/Program.cs

```diff
@@ -1,7 +1,40 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
+﻿using System.Reflection;
+
+//Strong password
+using (var stm = Assembly.GetExecutingAssembly ().GetManifestResourceStream ("MainRepository.Data.passwords.txt"))
+using (var sr = new StreamReader (stm)) {
+   while (!sr.EndOfStream) {
+      var result = sr.ReadLine ();
+      Console.WriteLine ($"The given {result} password is " + (isValidPassword (result) ? "Strong" : "Weak"));
+      Console.WriteLine ();
+   }
+}
+
+static bool isValidPassword (string s) {
+   string specialChar = "!@#$%^&*()-+";
+   if (s.Length < 6) {
+      Console.WriteLine ("Length of the password must be atleast 6.");
+      return false;
+   }
+   if (!s.Any (char.IsUpper)) {

```

Comment:
Instead of s.Any () for each condition, you could iterate the password string once to check for all the expected criteria.